custom/icds_reports/models/views.py

```diff
@@ -193,6 +193,8 @@ class ChildHealthMonthlyView(models.Model):
     current_month_wasting_v2 = models.TextField(blank=True, null=True)
     current_month_stunting_v2_sort = models.IntegerField(blank=True, null=True)
     current_month_wasting_v2_sort = models.IntegerField(blank=True, null=True)
+    contact_phone_number = models.TextField(blank=True, null=True)
+    phone_number = models.TextField(blank=True, null=True)
```

Comment:
what's the difference between contact_phone_number and phone_number? 
Please add a `help_text` to these as they are added so that they are documented

Reply:
I have added the help text.

Reply:
One is AWW phone number and the other is parent phone nubmer.